woocommerce/woocommerce
# Add coming soon template - minimal left image

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #52409 

This PR introduces a new 'coming soon' template. I believe we should limit the number of CSS styles added, as excessive styles could interfere with the default editor functionality. To address this, I focused on using only essential CSS, allowing users to customize the styles more effectively through the editor

![Image](https://github.com/user-attachments/assets/bb2687fc-eab8-40f6-8608-04c72af57033)

Besides, this PR also fixes an issue where users can't customize the social login button color (See recording below).

https://github.com/user-attachments/assets/78569eea-b7d0-41a7-b10a-698e967d2d56


P2: pfwcF5-13s-p2
Figma: 0BWMerqRR1f468EwC7g5tg-fi-3598_45708



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fresh site with  `coming-soon-newsletter-template` feature flag enabled.
2. Enable coming soon mode in site visibility settings.
3. Go to `Appearance > Editor > Templates > Coming soon`
4. You should be able to customize the color of the social login button using editor.
5. Choose `Coming Soon Minimal Left Image` template.
6. Confirm the template matches the design in Figma (0BWMerqRR1f468EwC7g5tg-fi-3598_45708).
7. Confirm that you can customize the styles, replace image, change the content, etc.
8. Save the template
9. Go to site frontend using a incognito tab to make sure the template is loaded correctly. The page should be responsive.
10. Go to `Appearance > Editor > Templates` and confirm the coming soon template preview shows the selected template.

Screenshots:

![Screenshot 2024-11-21 at 15 46 21](https://github.com/user-attachments/assets/78668565-cd3a-4274-a240-c95f9f2759f6)


![Screenshot 2024-11-21 at 14 29 12](https://github.com/user-attachments/assets/12899b7b-87f4-4feb-85e5-935a07463ad9)

![Screenshot 2024-11-21 at 15 17 14](https://github.com/user-attachments/assets/3f51a6c2-39b6-4f2c-978f-1ba48fcfa274)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
